app/code/OleksiiBezpoiasnyi/ControllerDemo/i18n/ru_RU.csv

```diff
@@ -0,0 +1,3 @@
+"Full name: %1","Полное имя: %1"
```

Comment:
@AlexeyBezpoyasny 
1) Avoid adding extra spaces to phrases
2) en_US.csv is missed